app/controllers/admin/schedules_controller.rb

```diff
@@ -7,10 +7,11 @@ class SchedulesController < Admin::BaseController
     load_resource :venue, through: :conference, singleton: true
 
     skip_before_filter :verify_authenticity_token, only: [:update]
-    layout 'schedule'
 
     def show
-      authorize! :update, @program.events.new
+      event = @program.events.new
```

Comment:
There is already an :update action. I think I don't understand what you mean :confused: